src/common/components/collapsible/Collapsible.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FunctionComponent } from 'react';
+
+import angleDownIcon from '../../../assets/icons/svg/angleDown.svg';
+import styles from './collapsible.module.scss';
+import Icon from '../icon/Icon';
+
+interface Item {
+  header: string;
+  body: string;
+}
+
+interface CollapsibleProps {
+  item: Item;
+}
+
+const Collapsible: FunctionComponent<CollapsibleProps> = ({ item }) => {
+  const [isOpen, toggleCollapsible] = React.useState(false);
+
+  return (
+    <>
+      <button
+        aria-expanded={isOpen}
```

Comment:
Good to add aria-expanded! 
and I think we need to add an aria-label too  if we want screen reader users to be able to know what this button does?